CI/physmon/physmon.py

```diff
@@ -298,9 +299,11 @@
                     sigmaScattering=5,
                     radLengthPerSeed=0.1,
                     minPt=500 * u.MeV,
-                    bFieldInZ=1.99724 * u.T,
                     impactMax=3 * u.mm,
                 ),
+                SeedFinderOptionsArg(
+                    bFieldInZ=1.99724 * u.T,
+                ),
```

Comment:
```suggestion
                SeedFinderOptionsArg(bFieldInZ=1.99724 * u.T),
```

I think the formatter is just unfolding this because of the `,`

Reply:
Thanks, this is addressed